Traceback (most recent call last):
  File "/seamless/seamless/core/transformation.py", line 251, in _execute
    result, logs = await self._execute_local(
  File "/seamless/seamless/core/transformation.py", line 392, in _execute_local
    validate_environment(env)
  File "/seamless/seamless/core/environment.py", line 162, in validate_environment
    raise ValueError("Environment error:\n" + err)
ValueError: Environment error:
Image:
  Cannot execute code locally: current image is 'rpbs/seamless', whereas 'nonsense' is required


Status: OK
<Silk: 112 >
